addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,490 @@
+  box-shadow: 0 0 0 0.125rem var(--editorDarkMode-primary-transparent35, hsl(260deg 60% 60% / 35%));
+}
+[class*="gui_body-wrapper_"] [class*="sprite-selector-item_is-selected"],
+[class*="gui_body-wrapper_"] [class*="stage-selector_is-selected_"] {
+  box-shadow: 0px 0px 0px 2px var(--editorDarkMode-primary-transparent35, hsl(260deg 60% 60% / 35%));
+}
+[class*="input_input-small"],
+[class*="sprite-info_larger-input"] input {
+  width: 2.5rem;
+}
+
+[class*="sprite-selector_scroll-wrapper"] {
+  height: calc(100% - 4.75rem);
+}
+[class*="sprite-selector_scroll-wrapper"]::-webkit-scrollbar-thumb {
+  background-color: var(--editorDarkMode-selector-compactScrollbar, #c1c8d1);
+}
+[class*="sprite-selector_items-wrapper"] {
+  padding: 0.125rem;
+}
+[class*="sprite-selector_sprite-wrapper"] {
+  margin: 0.125rem;
+}
+[class*="sprite-selector-item_delete-button"] {
+  top: -0.25rem;
+}
+[dir="ltr"] [class*="delete-button_delete-button_"] {
+  right: -0.25rem;
+}
+[dir="rtl"] [class*="delete-button_delete-button_"] {
+  left: -0.25rem;
+}
+[class*="delete-button_delete-button-visible"] {
+  width: 1rem;
+  height: 1rem;
+  box-shadow: 0 0 0 1px var(--editorDarkMode-primary-transparent35, hsl(260deg 60% 60% / 35%));
+}
+[class*="delete-button_delete-icon"] {
+  display: none;
+}
+[class*="delete-button_delete-button-visible"]::after {
+  content: "";
+  position: absolute;
+  display: block;
+  top: 25%;
+  left: 25%;
+  width: 50%;
+  height: 50%;
+  background-image: url("../../images/cs/close-s3.svg");
+  transform: rotate(45deg);
+  filter: var(--editorDarkMode-primary-filter, none);
+}
+img[class*="sprite-selector-item_sprite-image"] {
+  max-width: 56px;
+  max-height: 32px;
+}
+
+[class*="action-menu_main-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+[class*="action-menu_main-button"] img {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+[class*="action-menu_more-buttons-outer"] {
+  width: 2rem;
+  bottom: 2rem;
+  padding-bottom: calc(2rem / 2);
+  margin: 0 0 calc(2rem / -2) 0;
+  border-top-left-radius: 2rem;
+  border-top-right-radius: 2rem;
+}
+button[class*="action-menu_more-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+
+/* Project player */
+.sa-body-editor [class*="stage-header_stage-menu-wrapper_"] {
+  padding-top: 0.25rem;
+  padding-bottom: 0.25rem;
+  height: 2rem;
+}
+.sa-body-editor [class*="green-flag_green-flag_"],
+.sa-body-editor .pause-btn,
+.sa-body-editor [class*="stop-all_stop-all_"],
+.sa-body-editor [class*="stage-header_stage-button_"] {
+  padding: 0.25rem;
+  width: 1.5rem;
+  height: 1.5rem;
+}
+.sa-body-editor [class*="stage-header_stage-button_"] {
+  padding: calc(0.25rem - 1px);
+}
+.sa-body-editor [class*="green-flag_green-flag_"],
+.sa-body-editor .pause-btn {
+  margin-inline-end: 0.125rem;
+}
+.sa-stage-hidden [class*="stage-header_stage-size-row"] {
+  height: 2rem;
+}
+[class*="gui_stage-and-target-wrapper_"] [class*="stage-wrapper_full-screen_"] {
+  top: 2rem;
+}
```

Comment:
This selector intentionally targets fullscreen.

Reply:
Commit https://github.com/ScratchAddons/ScratchAddons/commit/3bd97f62f2e2f850e089fe709fbe90f78173d315

Reply:
I removed this CSS rule.